bigbluebutton-html5/imports/api/breakouts-history/index.js

```diff
@@ -0,0 +1,9 @@
+import { Meteor } from 'meteor/meteor';
+
+const BreakoutsHistory = new Mongo.Collection('breakouts-history');
+
+if (Meteor.isServer) {
+  BreakoutsHistory._ensureIndex({ meetingId: 1 });
```

Comment:
I believe I we modified all other index.js to contain a tweak for reconnection from @Tainan404 
Edit - looks like we have not merged https://github.com/bigbluebutton/bigbluebutton/pull/13574 yet. We need to be careful here with the order of the merges to ensure this collection does not slip